xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-test-pageobjects/src/main/java/org/xwiki/appwithinminutes/test/po/AppWithinMinutesHomePage.java

```diff
@@ -31,11 +31,6 @@
  */
 public class AppWithinMinutesHomePage extends ViewPage
 {
-    /**
-     * The title of the AppWithinMinutes home page.
-     */
-    public static final String TITLE = "App Within Minutes";
```

Comment:
That variable was used just in [these](https://github.com/gabrielasmeria/xwiki-enterprise/commit/339a00d84f3753b956dcd9caa13abbcf7113aa73) 2 functional tests to go to AWM's homepage, by clicking "App Within Minutes" from the breadcrumb.
And in order to have the current wiki homepage as a parent for the newly created app, I deleted "App Within Minutes" element from the breadcrumb (which was the parent for the newly created app). 
So, instead I used gotoPage() method to access AWM's homepage, as you can see in the link above.